libs/scripts/scripts.js

```diff
@@ -123,7 +123,7 @@ const config = {
   geoRouting: 'on',
   fallbackRouting: 'on',
   links: 'on',
-  imsClientId: 'milo',
+  imsClientId: 'fedsmilo',
```

Comment:
As usual, let's not forget to remove this before merging

Reply:
Done 👍